R/weight.R

```diff
@@ -44,3 +44,13 @@ weight <- function (x) {
     }
     return(x)
 }
+
+weightVariables <- function(x) {
```

Comment:
This function will need to be documented and exported in order for it to be useable.